scripts/deploy.sh

```diff
@@ -8,6 +8,8 @@ DOCKERHUB_ORG=$3
 
 # can be overridden via my_local_dev_vars
 PROJECT=${ASB_PROJECT}
+ASB_SCHEME="https"
```

Comment:
did we need this? what's the case where this is http?

Reply:
When `rcm` we use a different scheme `http`. I could actually remove this since it is the default in the template.